src/sqlfluff/cli/commands.py

```diff
@@ -403,6 +403,14 @@ def cli():
     """SQLFluff is a modular SQL linter for humans."""  # noqa D403
 
 
+cli.epilog = """\b\bExamples:\n
```

Comment:
I think it'd be cleaner to specify `epilog` as a keyword argument to `@click.group()`, i.e.:
```
@click.group(
    context_settings={"help_option_names": ["-h", "--help"]},
    epilog="""\b\bExamples:\n
  sqlfluff lint --dialect postgres .\n
  sqlfluff lint --dialect postgres --rules L042 .\n
  sqlfluff fix --dialect sqlite --rules L041,L042 src/queries\n
  sqlfluff parse --dialect sqlite --templater jinja src/queries/common.sql
""",
)
@click.version_option()
def cli():
    """SQLFluff is a modular SQL linter for humans."""  # noqa D403
```

Reply:
Cool! Done.